### ISSUE TITLE
userStoppedTypingHandler keeps calling after user stops typing

### ISSUE THREAD

**Issue by Prabu-raj** [apasccon/SearchTextField#77]
userStoppedTyping handler keeps invoking after the user stops typing the text.

        hospitalNameTextField.startVisible = true
        hospitalNameTextField.startVisibleWithoutInteraction = true
        
        hospitalNameTextField.userStoppedTypingHandler =
        {
            if let criteria = self.hospitalNameTextField.text
            {
                if criteria.characters.count > 0 && criteria.characters.count % 3 == 0
                {
                    self.hospitalNameTextField.showLoadingIndicator()
                    self.searchForHospitalNames(searchText: criteria)
                    {
                        results in
                        self.hospitalNameTextField.filterItems(results)
                        self.hospitalNameTextField.stopLoadingIndicator()
                    }
                }
            }
        }
        
        hospitalNameTextField.theme = SearchTextFieldTheme.lightTheme()
        
        hospitalNameTextField.itemSelectionHandler =
        {
            item, itemPosition in
            self.hospitalNameTextField.text = item[itemPosition].title
        }

**Comment by viswanath3344** [comment]
Yes. I am also getting same issue. Just needed like send request when user tap the key board.
